soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -50,7 +50,11 @@ void z_intel_adsp_ipc_isr(const void *devarg)
 
 		regs->tdr = INTEL_ADSP_IPC_BUSY;
 		if (done && !IS_ENABLED(CONFIG_SOC_INTEL_CAVS_V15)) {
-			regs->tda = INTEL_ADSP_IPC_DONE;
+			#ifdef CONFIG_SOC_SERIES_INTEL_ACE
+				regs->tda = INTEL_ADSP_IPC_ACE1X_TDA_DONE;
```

Comment:
ACE sets TDA==0, just like it did before!

Reply:
Yes, but not regs->ida! ;-)

Reply:
But... the code in this patch doesn't write to IDA anywhere?  I'm still confused.  Is it possible you pushed the wrong branch or something?